ResearchKit/Consent/ORKConsentSignatureFormatter.m

```diff
@@ -58,7 +59,10 @@ - (NSString *)HTMLForSignature:(ORKConsentSignature *)signature {
             if (signature.familyName) {
                 [names addObject:signature.familyName];
             }
-            nameStr = [names componentsJoinedByString:@"&nbsp;"];
+            if (ORKCurrentLocalePresentsFamilyNameFirst()) {
+                names = [[[names reverseObjectEnumerator] allObjects] mutableCopy];
+            }
+            nameStr = [names componentsJoinedByString:nameStr];
```

Comment:
It would be nice to make the separator explicit, just use:
`nameStr = [names componentsJoinedByString:@"&nbsp;"];`